### DISCUSSION TITLE
hexo deploy 后显示成功, 部署的文件也确定上传成功到github, 本地预览http://localhost:4000/也是有的, 但博客网页上没有更新

### DISCUSSION THREAD

**Discussion by CoderTitan** [hexojs/hexo#5029] [Question / Answer] [answered]
## Information

hexo deploy 后显示成功, 部署的文件也确定上传成功到github, 本地预览http://localhost:4000/也是有的, 但博客网页上没有更新

## Environment & Settings

**Node.js & npm version**

```
npm   v8.12.2
node  v16.15.1
```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
"hexo": "6.2.0",
"next": "8.12.2",
```

**Your package.json `package.json`**

```
"hexo": {
    "version": "6.2.0"
  },
  "dependencies": {
    "hexo": "^6.2.0",
    "hexo-abbrlink": "^2.2.1",
    "hexo-deployer-git": "^3.0.0",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-baidu-sitemap": "^0.1.9",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-feed": "^3.0.0",
    "hexo-generator-index": "^2.0.0",
    "hexo-generator-search": "^2.4.3",
    "hexo-generator-searchdb": "^1.4.0",
    "hexo-generator-sitemap": "^3.0.1",
    "hexo-generator-tag": "^1.0.0",
    "hexo-next-darkmode": "^0.1.10",
    "hexo-renderer-ejs": "^2.0.0",
    "hexo-renderer-marked": "^5.0.0",
    "hexo-renderer-stylus": "^2.1.0",
    "hexo-server": "^3.0.0",
    "hexo-symbols-count-time": "^0.7.1",
    "hexo-theme-landscape": "^0.0.3"
  }
```

## hexo deploy  输出结果

INFO  Start processing
INFO  Files loaded in 918 ms
INFO  0 files generated in 5.15 s
INFO  Deploying: git
INFO  Clearing .deploy_git folder...
INFO  Copying files from public folder...
INFO  Copying files from extend dirs...
位于分支 master
无文件要提交，干净的工作区
枚举对象中: 1116, 完成.
对象计数中: 100% (1116/1116), 完成.
使用 8 个线程进行压缩
压缩对象中: 100% (380/380), 完成.
写入对象中: 100% (622/622), 149.30 KiB | 1.82 MiB/s, 完成.
总共 622（差异 360），复用 0（差异 0），包复用 0
remote: Resolving deltas: 100% (360/360), completed with 241 local objects.
To https://github.com/CoderTitan/CoderTitan.github.io.git
   e311117..7914983  HEAD -> master
分支 'master' 设置为跟踪 'https://github.com/CoderTitan/CoderTitan.github.io.git/master'。
INFO  Deploy done: git

**Comment by renbaoshuo** [comment]
您的部署分支是不是设置错误了，看您的仓库应该部署到 `main` 分支的。

**Comment by stevenjoezhang** [comment, marked as answer]
请检查 GitHub Pages 配置，也可能是缓存问题
